### PR TITLE
Document what's a correctly-formatted CHANGELOG 

### PR DESCRIPTION
**Pull request by ivoanjo**
Fixes dblock/danger-changelog#40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,158 @@ It can make sure, for example, that changes are attributed properly, have a vali
...
+
+✅ Under each title, you can have changelog entries in the format: `* {optional link to GitHub pull request that must be followed by a colon} {a single sentence of text description} - {author github link}.`:
+
+```markdown
+* Entry without a PR link - [@ivoanjo](https://github.com/ivoanjo).
+* [#1](https://github.com/dblock/danger-changelog/pull/1): Entry with a PR link - [@dblock](https://github.com/dblock).
+* Yes, this is a weird entry, but it's still valid! Also Beyoncé is welcome here, as is 松本 行弘 and our nice emojis 👍 - [@ivoanjo](https://github.com/ivoanjo).
+```
+
+❌ Entry descriptions need to start with an uppercase A-Z letter
```
Comment (dblock) [inline README.md]: This looks like a bug. There's no reason why the first letter cannot be an uppercase Ñ. Maybe open an issue if you agree?

File: README.md [hunk 2]
```diff
@@ -6,6 +6,158 @@ It can make sure, for example, that changes are attributed properly, have a vali
...
+❌ GitHub pull request links need to be followed with a colon
+
+```markdown
+* [#1](https://github.com/dblock/danger-changelog/pull/1) Not valid - [@dblock](https://github.com/dblock).
+```
+
+❌ Only links to GitHub are allowed, and only using `https`
+
+```markdown
+* No GitLab - [@ivoanjo](https://gitlab.com/ivoanjo).
```
Comment (dblock) [inline README.md]: This looks like a bug too, we should support gitlab!

File: README.md [hunk 3]
```diff
@@ -6,6 +6,158 @@ It can make sure, for example, that changes are attributed properly, have a vali
...
+* Did stuff. Then more stuff - [@ivoanjo](https://github.com/ivoanjo).
+```
+
+❌ A dash needs to separate the text description from the author
+
+```markdown
+* Missing a dash here ➡️ [@ivoanjo](https://github.com/ivoanjo).
+```
+
+❌ An entry must end with a colon
```
Comment (dblock) [inline README.md]: With a period, not a colon.

**Comment by dblock** [comment]
Awesome! Only needs fixing the entries must end with a period.

File: README.md (hunk 3 above)
Comment (ivoanjo) [reply README.md]: Oops, temporary insanity :see_no_evil:

File: README.md (hunk 2 above)
Comment (ivoanjo) [reply README.md]: I'll report it 👍

File: README.md (hunk 1 above)
Comment (ivoanjo) [reply README.md]: Yes, I think it makes sense to allow all scripts, will report

Comment (ivoanjo) [reply README.md]: Somehow my testing was wrong (too much copy paste); I've re-tested and `Ñ` indeed works, so I've replaced it with a Japanese example.

File: README.md [hunk 4]
```diff
@@ -6,6 +6,158 @@ It can make sure, for example, that changes are attributed properly, have a vali
...
+* Entry without a PR link - [@ivoanjo](https://github.com/ivoanjo).
+* [#1](https://github.com/dblock/danger-changelog/pull/1): Entry with a PR link - [@dblock](https://github.com/dblock).
+* Yes, this is a weird entry, but it's still valid! Also Beyoncé is welcome here, as is 松本 行弘 and our nice emojis 👍 - [@ivoanjo](https://github.com/ivoanjo).
+```
+
+❌ Entry descriptions need to start with an uppercase A-Z letter
+
+```markdown
+* not valid - [@ivoanjo](https://github.com/ivoanjo).
+* いいえ - [@ivoanjo](https://github.com/ivoanjo).
```
Comment (dblock) [inline README.md]: How does it work in Japanese? Are there capital letters? If you think this should be legal, let's open an issue.

Comment (ivoanjo) [reply README.md]: They have three different scripts: Hiragana, Katakana, and Kanji; none of them have the concept of upper/lower (and none of them are accepted, I've tried).

Reported https://github.com/dblock/danger-changelog/issues/42

**Comment by dblock** [comment]
I'm merging this. Thanks! 

@ivoanjo If you want to contribute some code I could definitely use more collaborators/comaintainers of the gem!

**Pull request merged**